mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -493,12 +518,58 @@ message GenerateTxRequest {
 
     // Token id to use for the transaction.
     uint64 token_id = 7;
+
+    // List of SCIs to be added to the transaction
+    repeated SciForTx scis = 8;
 }
 
 message GenerateTxResponse {
     TxProposal tx_proposal = 1;
 }
 
+// Generate a transaction proposal object with mixed token types.
+// Notes:
+// - Sum of inputs needs to be greater than sum of outlays and fee.
+// - The set of inputs to use would be chosen automatically by mobilecoind.
+// - The fee field could be set to zero, in which case mobilecoind would choose a fee.
+// Right now that fee is hardcoded.
```

Comment:
We actually changed this and deleted the hard coded fee, it's not hard-coded anymore.
It will choose based on the network reported minimum fee, and if that doesn't exist, we have an error.